README.md

```diff
@@ -4,17 +4,12 @@
 
 This repo contains the source code for cross-platform [.NET Core](http://github.com/dotnet/core) command line toolchain. It contains the implementation of each command, the native packages for various supported platforms as well as documentation. 
 
-RC2 and Preview 1 bits
+RTM and Preview 2 bits
```

Comment:
There's a link to this section in the "Installers and Binaries" section (it says " please check out the section above") that I believe will need to change.